src/main/java/com/google/sps/data/PlayerDatabase.java

```diff
@@ -106,6 +125,31 @@ public int getEntityExperience() throws LoggedOutException {
     return Integer.parseInt(experienceString);
   }
 
+  public List<String> getEntityAllAccessoryIDs() throws LoggedOutException {
+    @SuppressWarnings("unchecked")
```

Comment:
Should we be checking for exceptions here?

Reply:
Yeah I didn't like the usage of this annotation but the casting/ supression of warnigns is what is documented in the GAE docs for getting multivalued attributes, see here: https://cloud.google.com/appengine/docs/standard/java/datastore/entity-property-reference